packages/api/src/apps/api/README.md

```diff
@@ -135,3 +135,7 @@ Note non synth historical prices are specified by erc20 address.
 ### tvl(addresses?: string[], currency: CurrencySymbol = "usd") => string
 
 Returns tvl of all supplied addresses in USD 18 decimals. If no addresses supplied, returns TVL of all known contracts.
+
+### getEmpStatsBetween(address: string, start = 0, end: number = Date.now(), currency: CurrencySymbol = "usd") => StatData[]
```

Comment:
would having a way to define the granularity be useful? for example, you _could_ have time-series data every second for the last year. requesting this would be an extremely taxing query.  if you could specify the granularity you could group things into say 10 mins bucket. 

by default, you could configure it to try an always return `n` number of data points where the granularity is dynamic depending on the time range requested. for example, you could say the query always returns 100 data points irrespective of the date range if no `granularity value is sent along.

This is not something that needs to be added in this PR, just something I wanted to bring up/ask about.

Reply:
As a second point, `Date.now()` is a milliseconds js timestamp. would it make sense to refactor this to take in a Unix timestamp? the rest of the repo works on Unix time so it might be more consistent to query things by seconds rather than milliseconds.

Reply:
> would having a way to define the granularity be useful? for example, you could have time-series data every second for the last year. requesting this would be an extremely taxing query. if you could specify the granularity you could group things into say 10 mins bucket.

I can definitely see this being useful and necessary for certain types of charts.  Will talk to tom about adding this as a feature. 

> As a second point, Date.now() is a milliseconds js timestamp. would it make sense to refactor this to take in a Unix timestamp? the rest of the repo works on Unix time so it might be more consistent to query things by seconds rather than milliseconds.

Good catch, currently all timestamps are stored in seconds internally. Its a bug and im querying things in MS once it hits the backend. Will need to fix this. 

As for the end user, i'm not sure what the right answer is in S vs MS.  To me its more intuitive to make queries with MS since most date manipulation classes give you this naturally, but I'm open to using seconds, just feels like an extra hoop for someone to jump through to get this.

Reply:
IMHO seconds is more natural but I do get your point that in js land everything is ms.

Reply:
i've changed it to seconds and we will see how it goes